[BUG] container image missing cert store

**Describe the bug**
Currently we can not use the container image, as it is missing any certificate store and therefore does not trust official certificates from the configured source (entraid)

**Log output**
```
time=2025-06-26T09:56:27.943Z level=INFO msg="initializing EntraID" package=entraid                                                                                                                                                           
time=2025-06-26T09:56:27.943Z level=INFO msg="processing EntraID groups" package=entraid.groups                                                                                                                                               
time=2025-06-26T09:56:45.796Z level=ERROR msg="could not get initial group result from EntraID" package=entraid.groups                                                                                                                        
panic: ClientSecretCredential: unable to resolve an endpoint: server response error:                                                                                                                                                          
     Get "https://login.microsoftonline.com/[redacted]/v2.0/.well-known/openid-configuration?": tls: failed to verify certificate: x509: certificate signed by unknown authority                                    
                                                                                                                                                                                                                                              
goroutine 1 [running]:                                                                                                                                                                                                                        
github.com/skuethe/grafana-oss-team-sync/internal/sources/entraid.ProcessGroups(0xc00007c820)                                                                                                                                                 
    /home/runner/work/grafana-oss-team-sync/grafana-oss-team-sync/internal/sources/entraid/groups.go:110 +0x631                                                                                                                               
github.com/skuethe/grafana-oss-team-sync/internal/sources.CallPlugin()                                                                                                                                                                        
    /home/runner/work/grafana-oss-team-sync/grafana-oss-team-sync/internal/sources/sources.go:33 +0x16c                                                                                                                                       
main.main()                                                                                                                                                                                                                                   
    /home/runner/work/grafana-oss-team-sync/grafana-oss-team-sync/main.go:70 +0x60f
```

**Expected behaviour**
We accept certificates from common certificate stores

**Additional context**

--- a/README.md
+++ b/README.md
@@ -201,7 +201,7 @@ If you want to build the project yourself, do the following
   ```
 3. Create the container image (adapt to your preferred tool for creating images)
   ```shell
-  podman build -t localhost/grafana-oss-team-sync:dev -f build/package/Dockerfile
+  podman build -t localhost/grafana-oss-team-sync:dev -f build/package/Dockerfile .
   ```
 
 <p align="right">( <a href="#top">Back to top</a> )</p>

--- a/build/package/Dockerfile
+++ b/build/package/Dockerfile
@@ -2,5 +2,6 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 FROM scratch
+COPY --from=docker.io/golang:alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY grafana-oss-team-sync /usr/bin/grafana-oss-team-sync
 ENTRYPOINT ["/usr/bin/grafana-oss-team-sync"]

--- a/deploy/local-tests_docker-compose.yaml
+++ b/deploy/local-tests_docker-compose.yaml
@@ -1,0 +1,25 @@
+# SPDX-FileCopyrightText: 2025 Sebastian Küthe and (other) contributors to project grafana-oss-team-sync <https://github.com/skuethe/grafana-oss-team-sync>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+services:
+  grafana:
+    image: docker.io/grafana/grafana-oss:latest
+    container_name: grafana
+    restart: unless-stopped
+    ports:
+     - '3000:3000'
+
+
+  grafana-oss-team-sync:
+    image: localhost/grafana-oss-team-sync:dev
+    container_name: grafana-oss-team-sync
+    build:
+      context: ../
+      dockerfile: build/package/Dockerfile
+    environment:
+      GOTS_HOST: "grafana:3000"
+      GOTS_RETRY: 10
+      GOTS_AUTHFILE: "/app/.env"
+      GOTS_CONFIG: "/app/configs/config.yaml"
+    volumes:
+      - '../:/app:Z'

